IdeventAPI/Controllers/ChipController.cs

```diff
@@ -59,7 +59,18 @@ public IActionResult GetById(int id)
             }
             return Ok(chip);
         }
-        
+
+        [HttpGet("Secret/{id}")]
```

Comment:
I stedet for Secret brug HashedId (bare for at være konsekvent i vores navngivning)

Reply:
Rettet